include/mbgl/gfx/backend.hpp

```diff
@@ -11,6 +11,7 @@ class Backend {
 public:
     enum class Type : uint8_t {
         OpenGL,
+        TYPE_MAX,
```

Comment:
What's this?

Reply:
The max type is used to determine the number of back-ends available. ProgramParameters uses this to size arrays for indexing shader source by back-end type.

Reply:
Ah so this one needs to be at the end when adding new backends.